<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormResCalc</class>
 <widget class="QWidget" name="FormResCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resistor Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_max_err">
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_vout"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Vout</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max. error (%):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_find_res">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_find_div">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_vref"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_expect_r"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Vref</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Expect resistence</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="checkBox_single">
       <property name="text">
        <string>Single resistor only</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_num_results">
       <property name="minimum">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Max No. of Results</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_vout</tabstop>
  <tabstop>lineEdit_vref</tabstop>
  <tabstop>pushButton_find_div</tabstop>
  <tabstop>spinBox_num_results</tabstop>
  <tabstop>checkBox_single</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>lineEdit_expect_r</tabstop>
  <tabstop>lineEdit_max_err</tabstop>
  <tabstop>pushButton_find_res</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
